tests/test_cases.json

```diff
@@ -420,5 +425,18 @@
         "fixed-encoding": "“lossy decoding�",
         "fixed": "\"lossy decoding�",
         "enabled": true
+    },
+    {
+        "label": "Synthetic, negative: Romanian word before a non-breaking space",
+        "original": "NICIODATĂ\u00a0",
+        "fixed": "NICIODATĂ\u00a0",
+        "enabled": true
+    },
+    {
+        "label": "Synthetic, false positive: the Sami name of a town in Lapland, in capital letters, followed by a non-breaking space",
+        "comment": "Related testing tells me I should worry about cases like this, though I haven't seen a real example",
+        "original": "SUOVÂKUOŠKÂ\u00a0",
+        "fixed": "SUOVÂKUOŠKÂ\u00a0",
+        "enabled": false
```

Comment:
Not that this would want to be fixed in this PR, but as a drive-by: I wonder if this file would be easier to read if `"enabled"` were considered `true` by default.

I also wonder if better than disabling the case would be, say, `"current": <whatever ftfy does now>`, which would override `"fixed"` but still let the test case run.  This way you would document what it's doing instead and be alerted if that behavior changed -- hopefully to the correct behavior, but presumably you want to find out anyway.

Reply:
I wanted the test case data to be reasonably explicit, but I can see why this would help -- it would make the disabled test cases useful for more than bookkeeping, and it'd be particularly desirable so we can notice if a case we're not fixing happens to become fixed.